### ISSUE TITLE
Using LLaVA-Med without webUI

### ISSUE THREAD

**Issue by stefanos831** [microsoft/LLaVA-Med#104]
is it a way to use llava-med through coding or api key without the webUI ? I want to use it as part of my research

**Comment by ZDH520a** [comment]
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:2 for open-end generation.为什么我会报这个错误，哥们能告诉我吗
